docs/deploying-airbyte/releases.md

```diff
@@ -0,0 +1,39 @@
+# Releases
+
+## Airbyte Core Releases
+
+Airbyte Core is any non-connector code. This includes the scheduler, workers, api, webapp, and the airbyte protocol.
```

Comment:
Airbyte Core is any non-connector code. This includes the scheduler, workers, api, web app, and the Airbyte protocol.